src/multivariate/composite.jl

```diff
@@ -0,0 +1,203 @@
+# Composite Multivariate distribution
+
+## Generic Composite multivariate distribution class
+
+module CompositeDistributions
+
+if !isdefined(:Distributions) using Distributions end
+
+import Base.length, Base.mean, Base.show
+import Distributions.params
+import Distributions.mean, Distributions.mode, Distributions.var, Distributions.cov
+import Distributions.entropy, Distributions.insupport
+import Distributions._logpdf, Distributions._pdf!
+import Distributions._rand!
+
+export AbstractCompositeContinuousDist, ContinuousMultivariateDistribution
```

Comment:
We don't abbreviate `Dist` in any type except the distributions of the F and t statistics. I would go for the more complete name that matches the majority of the other names.